lib/stat_tracker.rb

```diff
@@ -0,0 +1,24 @@
+class StatTracker
+  attr_reader :games, :teams, :game_teams
+
+  def initialize(games, teams, game_teams)
+    @games = games
+    @teams = teams
+    @game_teams = game_teams
+  end
+
+  def self.from_csv(locations)
+    # require "pry"; binding.pry
```

Comment:
All looks great! be sure to remove this commented out pry next time you are in the repo